airflow/models/variable.py

```diff
@@ -223,6 +223,7 @@ def rotate_fernet_key(self):
         if self._val and self.is_encrypted:
             self._val = fernet.rotate(self._val.encode('utf-8')).decode()
 
+    @staticmethod
```

Comment:
Wat. How did this ever work without this?

Reply:
Ough. Maybe that's why I was under the impression it wasn't working before - becuase it got the "class"  object passed as key !

Reply:
Though I also do not understand it :)